app/src/metrics.py

```diff
@@ -0,0 +1,31 @@
+'''
+The metric module provides metrics used to evaluate Fixed-time based simulation 
+Fuzzy-timed based one. 
+'''
+import traci
+
+def get_queue_length() -> int:
+    '''
+    Returns maximun queue lenght per steps.
+    '''
+    total = 0
+    lad_ids = traci.lanearea.getIDList()
+    for lad in lad_ids:
+        total += traci.lanearea.getLastStepVehicleNumber(lad)
+
+    return total
+
+
+def get_waiting_time() -> float:
+    '''
+    Returns total avreage waiting time per steps
```

Comment:
Je me demande quand même si ce n'est pas "The average waiting time" au lieu de "The  the total avreage waiting time"
```suggestion
    Returns the total avreage waiting time per step
```